platformio.ini

```diff
@@ -185,18 +161,25 @@ build_flags               = ${esp82xx_defaults.build_flags}
 ;                           -fexceptions
 ;                           -lstdc++-exc
 
-[core_pre]
-; *** Arduino Esp8266 core pre 2.6.x for Tasmota (recommended version, no known issues)
-platform                  = https://github.com/Jason2866/platform-espressif8266.git#Tasmota
+[core_2_6_1]
+; *** Esp8266 core for Arduino version 2.6.1 (for Windows, most Linux variants and Mac)
+; *** custom setup until the core 2.6.0 version is official released from PlatformIO crew
+platform                  = https://github.com/Jason2866/platform-espressif8266.git#core_2_6_1
 build_flags               = ${esp82xx_defaults.build_flags}
-                            -Wl,-Tesp8266.flash.1m.ld
+                            -Wl,-Teagle.flash.1m.ld
                             -O2
```

Comment:
It is unrelated to this PR but the -O2 does not have any effect in platform.io. It is overridden by a -Os by platformio, which is good.

Reply:
I've been wondering what the -O2 is doing there...